release-scripts/utils.py

```diff
@@ -30,7 +30,6 @@
 # Path names relative to the root of the project
 PATH_CALICOCTL_NODE = path.join("calicoctl", "calico_ctl", "node.py")
 PATH_CALICOCTL_INIT = path.join("calicoctl", "calico_ctl", "__init__.py")
-PATH_CALICONODE_BUILD = path.join("calico_node", "build.sh")
```

Comment:
Oops. didn't notice that `PATH_MAKEFILE` is included in `UPDATE_FILES_STATIC`. So this script *does* already correctly update those version pins over in the Makefile.

Nice one @robbrockbank  :)